scss/modules/_datatables.scss

```diff
@@ -128,3 +128,12 @@
   background-image: url('../img/i-sort-down--neutral.png');
   background-image: url('../img/i-sort-down--neutral.svg');
 }
+
+// Miscellaneous classes
+//
+// .is-incumbent      - If used, applied to candidate that currently holds the office in election tables
+
+.is-incumbent {
+  @include u-background-image('i-star--primary', 0% 50%);
```

Comment:
I found that when I changed the background-y position to 45%, the positioning appears a little above the line, which looks a little more centered.

![screen shot 2015-09-08 at 4 59 57 pm](https://cloud.githubusercontent.com/assets/1701077/9750321/f7231dcc-564a-11e5-8ea3-e04720a12e48.png)